api/src/services/notifications/groups/notify-admins-of-removed-user-service.ts

```diff
@@ -0,0 +1,56 @@
+import { Op } from "@sequelize/core"
+
+import { Group, User, Notification } from "@/models"
+
+import BaseService from "@/services/base-service"
+import { Notifications } from "@/services"
+
+export class NotifyAdminsOfRemovedUserService extends BaseService {
+  constructor(
+    private user: User,
+    private group: Group,
+    private currentUser: User
+  ) {
+    super()
+  }
+
+  async perform() {
+    await this.group.reload({ include: ["adminUsers"] })
```

Comment:
I've been leaning towards requiring the caller to provide all the correct includes via
```ts
if (isUndefined(this.group.adminUsers)) {
    throw new Error("Expected adminUsers association to be preloaded")
}
```
in all cases except background jobs.